build.gradle.kts

```diff
@@ -102,13 +108,13 @@ tasks.register<Test>("integTest") {
 /* RELEASING */
 tasks.register<Jar>("sourceJar") {
   from(sourceSets.main.get().allJava)
-  classifier = "sources"
+  archiveClassifier.set("sources")
```

Comment:
Dropping a note here on the switch to archiveClassifier for posterity: https://discuss.gradle.org/t/publishing-jars-with-classifiers-after-v5-0/30144/4

Reply:
Note: https://docs.gradle.org/7.6/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:classifier
is deprecated and replacement has `Property<String>` type which you cannot assign ([yet](https://github.com/gradle/gradle/pull/22240), but soon).